go.mod

```diff
@@ -61,7 +61,7 @@ require (
 require (
 	4d63.com/gocheckcompilerdirectives v1.2.1 // indirect
 	4d63.com/gochecknoglobals v0.2.1 // indirect
-	dario.cat/mergo v1.0.0 // indirect
+	dario.cat/mergo v1.0.1 // indirect
```

Comment:
**suggestion**
Since we don't use mergio anymore maybe this upgrade doesn't make sense.
```suggestion
	dario.cat/mergo v1.0.0 // indirect
```